### PR TITLE
feat(Secondary): Make initial component

### PR DESCRIPTION
**Pull request by michaeltaranto**
Add the ability to highlight a word in the middle of a sentence as `Secondary`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ after_success:
       if [ -n "$TRAVIS_PULL_REQUEST_SHA" ]; then
         npm run travis-deploy-once "npm run deploy-preview -- -d braid-design-system--${TRAVIS_PULL_REQUEST_SHA}.surge.sh";
         npm run travis-deploy-once "npm run post-commit-status";
+        npm run semantic-release -- --dry-run;
```
Comment (maraisr) [inline .travis.yml]: If not a PR, so a branch push, and its not master, try and semantic release?

Comment (markdalgleish) [reply .travis.yml]: This is just a temporary test for the new semantic-release config.

**Comment by michaeltaranto** [comment]
Perfect! For those playing at home, since merging this [release strategy PR](https://github.com/seek-oss/braid-design-system/pull/134) i used this PR to test the versioning outcomes behave as expected:

Before, this PR would have been a minor release (because its a `feat`):
> [10:38:42 PM] [semantic-release] › ℹ  Release note for version 0.1.0:
— See [travis log](https://travis-ci.org/seek-oss/braid-design-system/jobs/504955994#L689).

After, it's a patch release (as configured in https://github.com/seek-oss/braid-design-system/pull/134):
> [10:47:09 PM] [semantic-release] › ℹ  Release note for version 0.0.46:
— See [travis log](https://travis-ci.org/seek-oss/braid-design-system/builds/504958153#L694).

I'll now revert the testing changes in`.travis.yml`

File: .travis.yml [hunk 2]
```diff
@@ -11,6 +11,7 @@ after_success:
       if [ -n "$TRAVIS_PULL_REQUEST_SHA" ]; then
         npm run travis-deploy-once "npm run deploy-preview -- -d braid-design-system--${TRAVIS_PULL_REQUEST_SHA}.surge.sh";
         npm run travis-deploy-once "npm run post-commit-status";
+        npm run semantic-release -- --dry-run --no-ci;
```
Comment (markdalgleish) [inline .travis.yml]: You probably shouldn't check this in :trollface: 

(Hey, GitHub finally has a trollface emoji 😍)

Comment (michaeltaranto) [reply .travis.yml]: I really quite liked the dry run of having the version printed out along with the release notes in the build log. Felt clean. But yeah as mentioned, im pulling that out now 😄

File: .travis.yml (hunk 1 above)
Comment (maraisr) [reply .travis.yml]: Thanks guys! 🎉

**Pull request merged**